padd.sh

```diff
@@ -1415,7 +1429,7 @@ truncateString() {
 # https://unix.stackexchange.com/a/338844
 convertUptime() {
     # shellcheck disable=SC2016
-    eval "echo $(date -ud "@$1" +'$((%s/3600/24)) days, %H hours, %M minutes')"
+    eval "echo $($DATE -ud "@$1" +'$((%s/3600/24)) days, %H hours, %M minutes')"
```

Comment:
Suggestion:

Maybe you should use a more descriptive name for this variable, like `DATECMD`. 

When I first saw `DATE` here, I thought the variable was storing the actual date/time value.

Reply:
Great suggestion! I've updated.